vm/actor/src/util/chaos/types.rs

```diff
@@ -20,3 +24,43 @@ pub struct ResolveAddressResponse {
     pub address: Address,
     pub success: bool,
 }
+#[derive(Serialize_tuple, Deserialize_tuple)]
+pub struct SendArgs {
+    pub to: Address,
+    #[serde(with = "bigint_ser")]
+    pub value: TokenAmount,
+    pub method: u64,
```

Comment:
idrc because a test, but there is an alias for `MethodNum` that would be best to use here